schematic/manifest/generator.py

```diff
@@ -1547,35 +1546,42 @@ def get_manifest(
             return result
 
         # Generate empty template and optionally fill in with annotations
+        # if there is no existing manifest and use annotations is set to True, 
+        # pull annotations (in reality, annotations should be empty when there is no existing manifest)
         else:
             # Using getDatasetAnnotations() to retrieve file names and subset
             # entities to files and folders (ignoring tables/views)
-
             annotations = pd.DataFrame()
-            if self.is_file_based:
-                annotations = store.getDatasetAnnotations(dataset_id)
-
-            # if there are no files with annotations just generate an empty manifest
-            if annotations.empty:
-                manifest_url = self.get_empty_manifest(strict=strict)
-                manifest_df = self.get_dataframe_by_url(manifest_url)
-            else:
-                # Subset columns if no interested in user-defined annotations and there are files present
-                if self.is_file_based and not self.use_annotations:
-                    annotations = annotations[["Filename", "eTag", "entityId"]]
-
-                # Update `additional_metadata` and generate manifest
-                manifest_url, manifest_df = self.get_manifest_with_annotations(annotations, sheet_url=sheet_url, strict=strict)
+            if self.use_annotations:
+                if self.is_file_based:
+                    annotations = store.getDatasetAnnotations(dataset_id)
+                    # in the old code, we take a subset of columns if no interested in user-defined annotations and there are files present
+
+                # if there are no files with annotations just generate an empty manifest
+                if annotations.empty:
+                    manifest_df = self.get_dataframe_by_url(empty_manifest_url)
+
+            # else:
```

Comment:
Yeah I don't think we need this part. Is the `eTag` is just a hash for the annotations?

Reply:
Yeah I think so.. I agree we could remove that part. I removed it in the PR here: https://github.com/Sage-Bionetworks/schematic/pull/1279